combat-trainer.lic

```diff
@@ -2183,6 +2193,33 @@ class GameState
     end
   end
 
+  def tactics_analyze?
```

Comment:
The overlap is due to me basing the method off `perform_analyze?`, but they're sill fairly different in function.